modules/openapi-generator/src/main/resources/php-symfony/api.mustache

```diff
@@ -62,7 +62,7 @@ interface {{classname}}
      * @param  integer $responseCode     The HTTP response code to return
      * @param  array   $responseHeaders  Additional HTTP headers to return with the response ()
      *
-     * @return {{{vendorExtensions.x-comment-type}}}
+     * @return \{{{vendorExtensions.x-comment-type}}}
```

Comment:
What about fixing the issue by updating `vendorExtensions.x-comment-type` instead?

ref: https://github.com/OpenAPITools/openapi-generator/blob/60dcf8613f6b75b606757c4a991fe018ad10ee99/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpSymfonyServerCodegen.java#L421

Reply:
Thats a good suggestion, thanks @wing328. I'll take a look at that. I do not have much experience with this codebase, so this seemed the easiest way for now. But updating the method better.

Reply:
@wing328 it took me some time before I was able to compile everything, but I have changed this. Added a `"\\"` to the required lines. 

I also took the liberty to change the hardcoded `integer` to `int`